test/specs/bug/adjacent_lists.md

```diff
@@ -1 +0,0 @@
-*foo __bar *baz bim__ bam*
```

Comment:
Why was this test removed? Is it failing with this PR?

Reply:
nvm I see this was an accidental addition.

Reply:
That was a test I added playing around trying to debug my program and accidentally committed it to the PR.